backend/consensus/worker_service.py

```
@@ -352,7 +367,9 @@ async def health_check():
                 else:
                     hours = minutes // 60
                     tx["blocked_at"] = f"{hours}h ago"
-            except:
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)